### PR TITLE
NXP-19847: Fix GenericThreadedImportTask transaction rollback (7.10)

### PR DESCRIPTION
**Pull request by efge**

### CODE REVIEW

File: nuxeo-importer-core/src/main/java/org/nuxeo/ecm/platform/importer/base/GenericThreadedImportTask.java [hunk 1]
```diff
@@ -360,19 +361,26 @@ public synchronized void run() {
...
             notifyImportError();
         } finally {
             log.info("End of task");
-            if (session != null) {
-                session.close();
-                session = null;
-            }
-            synchronized (this) {
-                isRunning = false;
+            try {
```
Comment (ataillefer) [inline nuxeo-importer-core/src/main/java/org/nuxeo/ecm/platform/importer/base/GenericThreadedImportTask.java]: Why not use try-with-resources when opening the session?

Comment (efge) [inline nuxeo-importer-core/src/main/java/org/nuxeo/ecm/platform/importer/base/GenericThreadedImportTask.java]: Indeed that's a worthwhile refactoring but for this ticket I wanted the minimal fix possible.

**Comment by ataillefer** [comment]
:+1:

**Comment by atchertchian** [comment]
:+1:

**Pull request merged**